### PR TITLE
feat: support the deepmd-kit v3

### PR DESCRIPTION
**Pull request by zjgemi**

### CODE REVIEW

**Review by wanghan-iapcm — COMMENTED** [review]
could you please provide an example input script for the deepmd-kit v3 dpgen?

File: dpgen2/entrypoint/submit.py [hunk 1]
```diff
@@ -466,6 +470,14 @@ def workflow_concurrent_learning(
         ]
         upload_python_packages = _upload_python_packages
 
+    valid_data = config["inputs"]["valid_data_sys"]
+    if valid_data is not None:
+        valid_data_prefix = config["inputs"]["valid_data_prefix"]
+        if valid_data_prefix is not None:
+            valid_data = [os.path.join(valid_data_prefix, ii) for ii in valid_data]
```
[the remark this answers is not in the document]
Comment (zjgemi) [reply dpgen2/entrypoint/submit.py:479]: Updated.

File: dpgen2/entrypoint/submit.py [hunk 2]
```diff
@@ -517,15 +530,36 @@ def workflow_concurrent_learning(
...
+        head = config["inputs"]["head"]
+        multi_init_data = config["inputs"]["multi_init_data"]
+        init_data = []
+        multi_init_data_idx = {}
+        for k, v in multi_init_data.items():
+            sys = v["sys"]
+            if isinstance(sys, str):
+                sys = expand_sys_str(sys)
+            if v["prefix"] is not None:
+                sys = [os.path.join(v["prefix"], ii) for ii in sys]
```
[the remark this answers is not in the document]
Comment (zjgemi) [reply dpgen2/entrypoint/submit.py]: Updated.

**Comment by zjgemi** [comment]
> could you please provide an example input script for the deepmd-kit v3 dpgen?

Examples has been added in [https://github.com/zjgemi/dpgen2/tree/deepmd-v3/examples/water](https://github.com/zjgemi/dpgen2/tree/deepmd-v3/examples/water).

**Review by wanghan-iapcm — APPROVED** [review]

**Pull request merged**